test/cli/package-import-parent-package-conflict/parent/child/__package.rb

```diff
@@ -0,0 +1,5 @@
+# typed: strict
+
+class Parent::Child < PackageSpec
+  import Parent
```

Comment:
The issue, as I understand it, is that this `import` statement defines a constant `Parent` that is equal to the module `Parent`. But then you can't define `Parent::Child`... right?

Reply:
Yes, you would get a redefinition error in the namer. Often, that error is not easy to parse. Therefore, we catch it earlier and disallow the alias completely.